src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -50,7 +49,7 @@ public void AddComponent<TComponent>(string domElementSelector)
         /// </summary>
         /// <param name="componentType">The type of the component.</param>
         /// <param name="domElementSelector">A CSS selector that uniquely identifies a DOM element.</param>
-        public void AddComponent(Type componentType, string domElementSelector)
+        public async void AddComponent(Type componentType, string domElementSelector)
```

Comment:
@SteveSandersonMS I think we should simply remove all sync versions of this. As we spoke about in the past, if you don't want to handle errors nor wait for it to finish, don't await the Task

Reply:
Sounds good to me.